src/plugins/irc-events/connection.js

```diff
@@ -19,9 +19,9 @@ module.exports = function(irc, network) {
 		}));
 	});
 
-	irc.on("socket close", function() {
+	irc.on("close", function() {
 		network.channels[0].pushMessage(client, new Msg({
-			text: "Disconnected from the network."
+			text: "Disconnected from the network, and will not reconnect."
```

Comment:
Is there any addition `reason` or something we can give to the user? It seems very frustrating otherwise.